### DISCUSSION TITLE
Multiple buttons in a form do not work

### DISCUSSION THREAD

**Discussion by goddas** [livewire/livewire#1751] [1. Help]
### Description
I have a simple form component with the submit button and another `type=button` button. The later does not fire when inside the form.

### Exact steps to reproduce
The `button type="button"` does not fire when inside the form, works fine outside.

### Stripped-down, copy-pastable code snippets
```
 <form action="" method="get" wire:submit.prevent="search">
                        <div class="form-group mt-0">
                            <input wire:model.defer="searchQuery" type="search" name="search" >
                            @if(empty($searchQuery))
                                <button type="submit" >search</button>
                            @else
                                <button type="button" wire:click="clearSearch">Clear</button>
                            @endif
                        </div>
                    </form>
```
**Context**
- Livewire version: 2.2.9
- Laravel version: laravel/framework 8.8
- Browser: tested only in chrome

**Comment by goddas** [comment]
**Workaround**: use an 'a' tag and style it as a button.

**Comment by MichaelBrauner** [comment]
I think this is because u have set  `wire:submit.prevent` and the button want's to submit the form - but it is prevented.
Maybe wire:click.prevent on the button would also work. Or the <a> tag, yes.

**Comment by goddas** [comment]
> I think this is because u have set `wire:submit.prevent` and the button want's to submit the form - but it is prevented.
> Maybe wire:click.prevent on the button would also work. Or the tag, yes.

By default, only buttons with `type="submit"` submit the form. `type="button"` is specifically so that you can have multiple buttons in a form. So this is definitely a bug, because in this case `type="button"` button is not able to do anything
